tests/testYearWrap.py

```diff
@@ -26,13 +26,19 @@
 import unittest
 import lsst.utils.tests
 import lsst.ctrl.stats.records as recordslib
-from datetime import date
+import datetime
+from dateutil import tz, parser
 from lsst.ctrl.stats.reader import Reader
 
 
 def setup_module(module):
     lsst.utils.tests.init()
 
+def tolocal(dateVal):
```

Comment:
Duplicating this in every files has a bad "code smell"... I think I'd prefer a separate file for it imported into each test.